src/rules/pylint/rules/use_from_import.rs

```diff
@@ -31,11 +37,19 @@ pub fn use_from_import(checker: &mut Checker, alias: &Alias) {
     if name != asname {
         return;
     }
-    checker.diagnostics.push(Diagnostic::new(
+    let mut diagnostic = Diagnostic::new(
         ConsiderUsingFromImport {
             module: module.to_string(),
             name: name.to_string(),
         },
         Range::from_located(alias),
-    ));
+    );
+    if checker.patch(diagnostic.kind.rule()) {
+        diagnostic.amend(Fix::replacement(
+            format!("from {module} import {asname}"),
+            import_stmt.location,
+            alias.end_location.unwrap(),
```

Comment:
Should this not be `import_stmt.end_location.unwrap()`?

Reply:
OK, changed. it should be the same.
I also fixed a special case when there are multiple imports in that import statement.